pom.xml

```diff
@@ -693,6 +694,11 @@
         <artifactId>kerby-util</artifactId>
         <version>${kerby.version}</version>
       </dependency>
+      <dependency>
```

Comment:
if we add new dependencies, please specify the license governing the usage of this license to ensure it respects our license model. in most cases, these common dependencies are apache 2.0, but we want to confirm before merging